modules/cpuTasks/cpuTasks.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Retrieves stats about layouts, style recalcs and JS execution
+ *
+ * Metrics from https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pagemetrics
+ */
+"use strict";
+
+module.exports = function (phantomas) {
+  // Converts seconds into milliseconds
+  function ms(value) {
+    return Math.round(value * 1000);
+  }
+
+  phantomas.on("metrics", (metrics) => {
```

Comment:
I hope that one day we'd be able to use benefits of TypeScript here :slightly_smiling_face: 

https://github.com/macbre/phantomas/issues/830